rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/search.aggregation/40_range.yml

```diff
@@ -7,420 +7,259 @@ setup:
-
-  - do:
-      index:
-        index: test
-        id: 2
-        body: { "date" : "315561600000" }
-
-  - do:
-      index:
-        index: test
-        id: 3
-        body: { "date" : "631180800000" }
-
-  - do:
-      index:
-        index: test
-        id: 4
-        body: { "date" : "10000" }
-
-  - do:
-        index:
-          index: test
-          id: 5
-          body: { "ip" : "192.168.0.1" }
-
-  - do:
-      indices.refresh: {}
-
+"Double range default keyed response":
   - do:
       search:
         rest_total_hits_as_int: true
-        body:
-          aggs:
-            age_groups:
-              date_range:
-                field: date
-                missing: "0"
-                ranges:
-                - key: Generation Y
-                  from: '315561600000'
-                  to: '946713600000'
-                - key: Generation X
-                  from: "200000"
-                  to: '315561600000'
-                - key: Other
-                  to: "200000"
-
-  - match: { hits.total: 5 }
-
-  - length: { aggregations.age_groups.buckets: 3 }
-
-  - match: { aggregations.age_groups.buckets.0.key: "Other" }
-
-  - match: { aggregations.age_groups.buckets.0.doc_count: 2 }
-
-  - match: { aggregations.age_groups.buckets.1.key: "Generation X" }
-
-  - match: { aggregations.age_groups.buckets.1.doc_count: 1 }
-
-  - match: { aggregations.age_groups.buckets.2.key: "Generation Y" }
-
-  - match: { aggregations.age_groups.buckets.2.doc_count: 2 }
-
-
----
-"Date range unmapped with children":
-  - skip:
-      version: " - 7.9.99"
-      reason: Fixed in 7.10.0
-
-  - do:
-      indices.create:
-          index: test_a_unmapped
-          body:
-            settings:
-              number_of_shards: 1
-              number_of_replicas: 0
-  - do:
-      search:
-        index: test_a_unmapped
-        body:
-          size: 0
-          query:
-            terms:
-              animal: []
-          aggs:
-            date_range:
-              date_range:
-                field: date
-                ranges:
-                 - from: 2020-01-01T00:00:00Z
-              aggs:
-                sounds:
-                  cardinality:
-                    field: sound.keyword
+        body: {
+              "size": 0,
+              "aggs": {
+                "double_range": {
+                  "range": {
+                    "field": "double",
+                    "keyed": true,
+                    "ranges": [
+                      {
+                        "key": "first",
+                        "to": 50
+                      },
+                      {
+                        "key": "another",
+                        "from": 50,
+                        "to": 150
+                      },
+                      {
+                        "key": "last",
+                        "from": 150
+                      }
+                    ]
+                  }
+                }
+              }
+            }
```

Comment:
These'd be shorter in yaml. You know you can send yaml in curl if you want too. It works.

Reply:
I didn't know you could send yaml in curl, actually.  I'll have to try that the next time I'm messing around with it.  This is the result of me copy/pasting an existing test.  If you feel strongly about it, I'll clean it up, otherwise I don't think it's a big deal if it sits like this for a while.